src/ops/direnv/envrc.bash

```diff
@@ -6,6 +6,30 @@ function punt () {
     :
 }
 
+# move "origPreHook" "preHook" "$@";;
+move() {
+    srcvarname=$1 # example: varname might contain the string "origPATH"
+    # drop off the source variable name
+    shift
+
+    destvarname=$1 # example: destvarname might contain the string "PATH"
+    # drop off the destination variable name
+    shift
+
+    # like: export origPATH="...some-value..."
+    export "${@?}";
+
+    # set $original to the contents of the variable $srcvarname
+    # refers to
+    eval "$destvarname=\"${!srcvarname}\""
+
+    # mark the destvarname as exported so direnv picks it up
+    export "$destvarname"
+
+    # remove the export from above, ie: export origPATH...
+    unset "$srcvarname"
```

Comment:
I didn't know about `export -n` before. I guess it is equivalent to `unset` ? In that case I think `unset` might be better because it seems more common and simply says what it does :)